packages/headless/src/controllers/commerce/core/sub-controller/headless-sub-controller.ts

```diff
@@ -94,29 +150,73 @@ export function buildSearchAndListingsSubControllers(
   } = subControllerProps;
   return {
     ...buildBaseSubControllers(engine, subControllerProps),
+    /**
+     * Sorts the products.
+     *
+     * @param props - The sort properties.
+     * @returns The sorted products.
+     */
     sort(props?: SortProps) {
```

Comment:
Why do we need these functions documented? They should be covered by the documentation on the type itself (https://github.com/coveo/ui-kit/pull/3913/files#diff-70fb810df3b4d7646a54a4eb9c51a65a82fe48917db532f2018810b71ea340ecR56)